src/.vuepress/config.js

```diff
@@ -271,62 +269,62 @@ module.exports = {
     logo: '/logo.png',
     nav: [
       {
-        text: 'Docs',
-        ariaLabel: 'Documentation Menu',
+        text: 'Documentación',
+        ariaLabel: 'Menú de Documentación',
         items: [
           {
-            text: 'Guide',
+            text: 'Guía',
             link: '/guide/introduction'
           },
           {
-            text: 'Migration Guide',
+            text: 'Guía de Migración',
             link: '/guide/migration/introduction'
           },
           {
-            text: 'Style Guide',
+            text: 'Guía de Estilo',
             link: '/style-guide/'
           },
           {
-            text: 'Cookbook',
+            text: 'Recetario',
```

Comment:
Same wording change here 😄 
```suggestion
            text: 'Libro de Recetas',
```